otj-server-reactive/src/main/java/com/opentable/server/reactive/ReactiveServerJacksonConfiguration.java

```diff
@@ -0,0 +1,26 @@
+package com.opentable.server.reactive;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.springframework.boot.web.codec.CodecCustomizer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.codec.json.Jackson2JsonDecoder;
+import org.springframework.http.codec.json.Jackson2JsonEncoder;
+
+@Configuration
+public class ReactiveServerJacksonConfiguration {
+
+    @Bean
+    public CodecCustomizer otCodecCustomizer(final ObjectMapper objectMapper) {
```

Comment:
This approach doesn't work with `DelegatingWebFluxConfiguration`. I see now why you removed that import now from `@ReactiveServer` annotation. I think we need to figure out how to get this to work with both `DelegatingWebFluxConfiguration` _and_ `WebFluxAutoConfiguration`.